app/views/ctc/ctc_pages/_question.html.erb

```diff
@@ -14,4 +14,4 @@
       </div>
     </div>
   </div>
-</main>
+</div>
```

Comment:
[dust] I confess I don't understand this change -- which of the issues does this fix? I'm used to *introducing* `main` when possible as being more semantic, so on a surface level this surprises me.

Reply:
Yes, we should only have one main landmark per page. See https://github.com/newjersey/accessibility-pm/issues/1

Reply:
got it, thanks!